frontend/src/metabase/browse/components/EllipsifiedWithMarkdownTooltip.tsx

```diff
@@ -14,7 +16,14 @@ export const EllipsifiedWithMarkdownTooltip = ({
         </Markdown>
       }
     >
-      {children}
+      <Markdown
```

Comment:
:x: 

Tooltip is shown when there is no ellipsis

![image](https://github.com/user-attachments/assets/f1e33f44-2348-4976-a24a-6533b13a6d16)


I thought it's a bug in master, so here's a nice report :laughing: #47340

Reply:
This was a though one, and I might have done [something questionable](https://github.com/metabase/metabase/pull/47277/files#diff-d837076154adc8de87acd15299663110f0714b90ff1b02e600c704cccfe55b50R6-R21) to fix it.

The truncation detection does not work when there are block elements in the element that is being checked for truncation, so I resulted to making all elements inline.

Reply:
> I think it was intentional and that was the whole point of EllipsifiedWithMarkdownTooltip.

I struggle to see why we would want to render raw markdown as opposed to rendered markdown.
I do see why we might want to omit certain elements though to avoid confusion.

From the conversations [on this PR and it's linked Slack thread](https://github.com/metabase/metabase/pull/42456) I gather that this was done because it was deemed technically impossible to do the markdown rendering in the cell correctly with the truncation. A problem which I've solved in this PR.

@rafpaf Can you remember this was intentional or if it was incidental? Would rendering the markdown in the cell if we can be preferred to rendering raw markdown?

Reply:
Checked in again with @mazameli and he's happy with the changes:

> then I don’t think I see any issues. italics, links, bold, etc all seem chill, and if there are paragraph breaks I trust our default truncation to keep things sane.